textdescriptives/tests/test_coherence.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+import pytest
+import spacy
+
+from textdescriptives.components import Coherence
+
+
+@pytest.fixture(scope="function")
+def nlp_small():
+    nlp = spacy.load("en_core_web_sm")
+    nlp.add_pipe("textdescriptives.coherence")
+    return nlp
+
+
+@pytest.fixture(scope="function")
+def nlp_large():
+    nlp = spacy.load("en_core_web_lg")
+    nlp.add_pipe("textdescriptives.coherence")
+    return nlp
```

Comment:
Any reason for the large model? It will just slow down the tests

Reply:
not afterall no, the medium/large models have different kinds of embeddings than the small model so I just wanted to see if the code worked with both. 
Seems they both add the .vector attribute so let's remove the nlp_large model